src/main/java/com/diffblue/cover/annotations/generated/MethodUnderTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 Diffblue Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.diffblue.cover.annotations.generated;
+
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.RetentionPolicy.SOURCE;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates the method being tested in the annotated test method. Diffblue Cover attaches this
+ * annotation to all tests that it creates.
+ */
+@Retention(SOURCE)
+@Target(METHOD)
+public @interface MethodUnderTest {
```

Comment:
❓ `@MethodUnderTest` vs `@MethodsUnderTest` ? I'm slightly inclined to add the `s` since it supports multiple values... but I'm also aware that the singular case will be most common.

Reply:
I will change this to plural, now I see the ticket was defined like that as well